src/RequestListener.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace Opstalent\CrudBundle;
+
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\Form\FormInterface;
+use Symfony\Component\HttpKernel\Event\GetResponseEvent;
+use Symfony\Component\HttpKernel\KernelEvents;
+use Symfony\Component\HttpFoundation\Request;
+
+/**
+ * Class RequestListener
+ * @author Szymon Kunowski <szymon.kunowski@gmail.com>
+ * @package Opstalent\CrudBundle
+ */
+class RequestListener implements EventSubscriberInterface
+{
+    /**
+     * @var FormConfigResolver
+     */
+    protected $resolver;
+
+    /**
+     * @var FormFactory
+     */
+    protected $factory;
+
+    /**
+     * RequestListener constructor.
+     * @param FormConfigResolver $resolver
+     * @param FormFactory $factory
+     */
+    public function __construct(FormConfigResolver $resolver, FormFactory $factory)
+    {
+        $this->resolver = $resolver;
+        $this->factory = $factory;
+    }
+
+    /**
+     * @return array
+     */
+    public static function getSubscribedEvents()
+    {
+        return [
+            KernelEvents::REQUEST => [
```

Comment:
it could be `KernelEvents::REQUEST => ['handleForm', -255]` if it is only one listener here